Sources_v3/StreamChat/ChatClient.swift

```diff
@@ -394,6 +533,8 @@ extension ClientError {
 /// its `RequestEncoder`.
 extension _ChatClient: ConnectionStateDelegate {
     func webSocketClient(_ client: WebSocketClient, didUpdateConectionState state: WebSocketConnectionState) {
+        connectionStatus = .init(webSocketConnectionState: state)
```

Comment:
This will never run for passive clients so we'll always have `initializing` state, right?

Reply:
Yep. Does it make sense to you or ..? 😄

Reply:
Good point @b-onc. We should probably check for the config here and set the value to `notConnected` instead of `initialized`.

Reply:
fixed